rules/shared-structures.yaml

```diff
@@ -83,9 +83,8 @@ CAHOSP:
 	Capturable:
 	ExternalCapturable:
 		CaptureCompleteTime: 7
-	ProximityExternalCondition:
-		Condition: medic_heal
-		Range: 707c109
+	ExplodeWeapon@RepairInfantryGlobal:
```

Comment:
The idea was to move all SelfHealing to warheads with negative damage to allow stacking of the repair speed in case of multiple repair units / buildings.